### PR TITLE
Basic auth as in issue #8 + behat + phpspec tests

### PR DESCRIPTION
**Pull request by potfur**

### CODE REVIEW

**Comment by potfur** [comment]
@marcotroisi Please enable travis integration.

**Comment by marcotroisi** [comment]
Of course...

File: features/front/index.feature [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Feature: There should be something on front
```
Comment (marcotroisi) [inline features/front/index.feature:1]: but, where is the Behat-generated `Context` for this feature?

Comment (potfur) [inline features/front/index.feature:1]: ehm.... /features/bootstrap/FeatureContext.php

Comment (marcotroisi) [inline features/front/index.feature:1]: but this particular test is not there... or am I not seeing it?

Comment (potfur) [inline features/front/index.feature:1]: `Feature` is just a name describing suite of scenarios.
Methods in `FeatureContext` are case specific sentences used in those scenarios - like _I am authenticated with login "test" and password "test"_

Comment (marcotroisi) [inline features/front/index.feature:1]: Yes, I understand that ;) - but I do not see how `ensureUserWithCredentials` or `authenticatedWithCredentials` can actually test that I see "Hoppla!" when I visit _/_...

Comment (marcotroisi) [inline features/front/index.feature:1]: ok, clarified.

**Pull request merged**